### PR TITLE
Add Zcash Node

### PR DESCRIPTION
**Pull request by nfl0**
Example SDL for running Zcash Node on Zcash mainnet

### CODE REVIEW

**Comment by andy108369** [comment]
@nfl0 thank you for your PR!
Please update the root level README.md before we can merge it.

**Comment by nfl0** [comment]
Thanks for your feedback! I've made the necessary changes and believe the code is now ready for merge. Please let me know if there's anything else that needs to be addressed before we proceed.

**Comment by andy108369** [comment]
@nfl0 looks great! Please add the last bit - `signedBy` attributes, e.g. https://github.com/akash-network/awesome-akash/blob/b720cd5/MySQL/deploy.yaml#L29-L32 to your deployment SDL manifest.

Thank you!

File: README.md [hunk 1]
```diff
@@ -113,6 +113,7 @@ Awesome DeFi apps you can deploy on Akash
 - [Ethereum 2.0](Ethereum_2.0)
 - [POKT Network](pokt_network)
 - [Bitcoin Cash Node](bitcoincashnode)
+- [Zcash Node](zcashnode)
```
Comment (andy108369) [inline README.md:116]: Please rename "zcashnode" to "Zcash" or other way around so it matches the directory you are adding ("Zcash" directory)

**Review by andy108369 — CHANGES_REQUESTED** [review]

**Review by andy108369 — CHANGES_REQUESTED** [review]

File: Zcash/start.sh
[the remark this answers is not in the document]
Comment (Dimokus88) [reply Zcash/start.sh:1]: Hey @nfl0  !
You can look to the recommendation @andy108369 and resolve the branch conflict  ?

**Comment by AlirezaDarbandi** [comment]
> Example SDL for running Zcash Node on Zcash mainnet

**Comment by Dimokus88** [comment]
Hey @nfl0 !
Need resolve branch conflict, for merge.

**Comment by nfl0** [comment]
Thank you for bringing the issue to my attention. I am currently working on resolving it.

File: Zcash/start.sh
[the remark this answers is not in the document]
Comment (nfl0) [reply Zcash/start.sh:1]: Converted, see Dockerfile, thanks for recommendations.

**Comment by nfl0** [comment]
@Dimokus88 done

**Comment by nfl0** [comment]
I'm closing this PR as zcashd will be deprecated in the next few months and replaced by the Zcash Zebrad software, which is already in production: [Zebra](https://github.com/ZcashFoundation/zebra).

**Pull request closed without merging**